v5/ooapi.v5/Controllers/AcademicSessionsController.cs

```diff
@@ -52,7 +52,7 @@ public AcademicSessionsController(IConfiguration configuration, CoreDBContext db
     [Route("academic-sessions")]
     [ValidateModelState]
     [SwaggerOperation("AcademicSessionsGet")]
-    [SwaggerResponse(statusCode: 200, type: typeof(AcademicSessions), description: "OK")]
+    [SwaggerResponse(statusCode: 200, type: typeof(Pagination<AcademicSession>), description: "OK")]
```

Comment:
AcademicSessions erft al van Pagination<AcademicSession>, dus ik zou niet verwachten dat deze change noodzakelijk is.
Indien de swagger niet goed is moeten kijken of de inheritance wel wordt gesnapt. Ik zie nu bijvoorbeeld geen KnownTypes attributes op die classes.

Reply:
Dit geldt overigens ook voor de andere aanpassingen